### DISCUSSION TITLE
License change Country Metadata

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3928] [General]
[osm2lanes expressed interest](https://github.com/a-b-street/osm2lanes/issues/103) in using some of the data in `/res/country_metadata/`, which means this is going to be put and maintained in a separate repository. However, in the absence of any other information, that data is currently licensed under GPL-3 as the rest of the project. This license is not usable for non-GPL projects such as a-b-street or osmlanes for that matter.

**I think most uncomplicated for users of this data would be public domain / CC-0.** I hereby declare my contributions to the country metadata as public domain / CC-0.

To all people who contributed to the country metadata once, would a license change to CC-0 / public domain be acceptable to you? Just give a thumbs-up on this post then.

And if not, what would be acceptable? (E.g. CC-BY, CC-BY-SA? Write a comment)

- [x] _many_  @matkoniecz
- [x] _many_  @FloEdelmann
- 🚫 `orchardProduces.yml` @rugk 
- [x] `regularShoppingDays.yml` @andreasn 
- [x] `popularSports.yml` 
  - [x] @Etua 
  - [x]  @goldfndr 
  - [x]  @mykyta-nikiforov
  - [x]  @stefanb 
- [x] `popularReligions.yml`
  - [x] @stefanb 
  - [x] @flippipe 
  - [x] @Discostu36
- [x] `noStoppingSignStyle.yml` @Dimitar5555
- [x] `noParkingSignStyle.yml` @Dimitar5555
- [x] `additionalValidHousenumberRegex.yml` @1ec5 
- [x] `firstDayOfWorkweek.yml` @rugk

**Comment by rugk** [comment]
`orchardProduces.yml` is CC BY-NC-SA 3.0 IGO as it's generated from that source data, see https://github.com/rugk/crops-parser#legal-stuff and related discussion: https://github.com/rugk/crops-parser/issues/60

`firstDayOfWorkweek.yml` is fine of course (was anyway only a minor change)

**Comment by FloEdelmann** [comment]
For reference:

* https://github.com/streetcomplete/StreetComplete/commit/e5fd661b409378e42e44f481daac5831ce45e29e
* https://github.com/streetcomplete/StreetComplete/commit/4e78ee4973595a878c775a4b2ebc9457a0e68d9a
* https://github.com/streetcomplete/StreetComplete/commit/10cce92f0e20b937f93a913091e54205469dfd07
* https://github.com/streetcomplete/countrymetadata

**Comment by peternewman** [comment]
FWIW, roofsAreUsuallyFlat.yml came from https://docs.google.com/spreadsheets/d/1Oc4i2KsF-FyJ-Bxpu1dcbGK1Wrae7mHeg_CM2rpNBc8/edit#gid=1964714622 which based on the checker names listed there was created by the following:

- @bansa85
- @blueplasticbag
- @nagisa
- @naposm
- @michaelblyons
- @westnordost
- @matkoniecz
- @peternewman
- @bansan85

So they should probably thumbs up the original post too if they haven't...

@blueplasticbag only did one row which @bansan85 also covered so I think we're okay there.

**Comment by nagisa** [comment]
I don't particularly care the exact licensing my “contribution” to `roofsAreUsuallyFlat` ends up using, but more generally I'd like to point out that licensing such data as liberally as CC0/public-domain without any scheme to keep it synchronized between projects will only lead to unfortunate discrepancies in data over time.

My intuition that this data is more like the map data itself in how it wants to be handled. In particular at least the share-alike and keep-open clauses from the ODbL license seem like valuable properties for this sort of data.

**Comment by bansan85** [comment]
I'm not sure if I need to do something. But just to say it : I'm fine to change the license. I just want my data remains publicly accessible. It's better to have the less restricted license.
